mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/css/main.css

```diff
@@ -3,6 +3,10 @@
 /* Overridden aui tags
 */
 
+.aui header, .aui nav {
+	display: none;
+}
+
```

Comment:
Instead of hiding default rendered elements, don't include them on the page. To achieve this, override default template in your own theme.

Please send the theme and CSS in a separate PR from JavaScript changes.